spec/paperclip/storage/s3_spec.rb

```diff
@@ -379,6 +379,54 @@ def counter
     end
   end
 
+  context "An attachment that uses S3 for storage and has styles" do
+    before do
+      rebuild_model(
+        (aws2_add_region).merge({
+          storage: :s3,
+          styles: { thumb: ["90x90#", :jpg] },
+          bucket: "bucket",
+          s3_credentials: {
+            "access_key_id" => "12345",
+            "secret_access_key" => "54321" }
+          }))
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.